policyengine_us/variables/gov/states/oh/tax/income/credits/retirement_income/oh_lump_sum_retirement_credit.py

```diff
@@ -0,0 +1,57 @@
+from policyengine_us.model_api import *
+
+
+class oh_lump_sum_retirement_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Ohio Lump Sum Retirement Income Credit"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://codes.ohio.gov/ohio-revised-code/section-5747.055"  # (C, D, E)
+    defined_for = "oh_pension_based_retirement_income_credit_eligible"
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.oh.tax.income.credits.retirement
+
+        person = tax_unit.members
+        is_spouse = person("is_tax_unit_spouse", period)
+        is_head = person("is_tax_unit_head", period)
+
+        pension_income = person("pension_income", period)
+        age_head = tax_unit("age_head", period)
```

Comment:
can do all this at the person level and then multiply by head_or_spouse to avoid duplicating the code for head and spouse